test/unit/computed/transaction.spec.js

```diff
@@ -68,5 +68,43 @@ describe('Computed Transactions Unit Tests', () => {
       expect(tx.amountLabel, 'to match', /63[,.]67/);
       expect(tx.feeLabel, 'to match', /0[,.]57/);
     });
+
+    it('should limit transactions to last 100', () => {
+      store.transactions = [];
+      store.payments = [];
+      store.invoices = [];
+      const TRANSACTIONS_COUNT = 101;
+      const randomNumber = ({ min, max }) =>
+        Math.floor(Math.random() * (max - min + 1)) + min;
+      [...Array(TRANSACTIONS_COUNT)].forEach((v, i) => {
+        const transactionType = randomNumber({ min: 1, max: 3 });
+        let transaction = {
+          id: i,
+          amount: randomNumber({ min: 1, max: 999999 }),
+          date: new Date(),
+        };
+        if (transactionType === 1) {
+          transaction.type = 'bitcoin';
+          transaction.fee = randomNumber({ min: 250, max: 9999 });
+          transaction.confirmations = randomNumber({ min: 0, max: 12 });
+          transaction.status =
+            transaction.confirmations >= 6 ? 'confirmed' : 'unconfirmed';
+          store.transactions.push(transaction);
+        } else {
+          transaction.type = 'lightning';
+          transaction.status = randomNumber({ min: 0, max: 1 })
+            ? 'complete'
+            : 'in-progress';
+          if (transactionType === 3) {
+            transaction.fee = randomNumber({ min: 0, max: 250 });
+            store.payments.push(transaction);
+          } else {
+            store.invoices.push(transaction);
+          }
+        }
```

Comment:
Probably easier to just clone `store.transactions` 101 times. No need for all the test logic here...